backend/.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules/
 .env
+
+# Secrets
+credentials.csv
```

Comment:
interesting csv format for credentials?

Reply:
O right that's actually a backup in case we lose the AWS credentials in our .env. I'll probably remove this line in a future PR